examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -39,132 +45,148 @@
 # Authors: Elvis Dhomatob, <elvis.dohmatob@inria.fr>, Apr. 2014
 #          Virgile Fritsch, <virgile.fritsch@inria.fr>, Apr 2014
 #          Gael Varoquaux, Apr 2014
+#          Andres Hoyos-Idrobo, Apr 2017
+
+######################################################################
+# Loading the data
+######################################################################
```

Comment:
For consistency with the other titles in the example, you probably want the following change:
```suggestion
# --------------------------------
```